source/manual/renew-a-tls-certificate.html.md

```diff
@@ -6,52 +6,63 @@ layout: manual_layout
 parent: "/manual.html"
 ---
 
-This document covers how to renew Gandi wildcard TLS certificates (`*.publishing.service.gov.uk`, `*.staging.publishing.service.gov.uk`
-and `*.integration.publishing.service.gov.uk`) and `www.gov.uk`.
+## Renewing the certificate for www.gov.uk
 
-Credentials for the Fastly dashboard and Zendesk support sites are in the
-[2nd line password store](https://github.com/alphagov/govuk-secrets/blob/master/pass/2ndline/fastly).
+The TLS certificate for www.gov.uk is managed by Fastly. Fastly will open a support
+ticket when the certificate is due for renewal. This ticket will be picked up by
+GOV.UK Replatforming, who will co-ordinate with Fastly to renew the
+certificate.
+
+Note that the www.gov.uk certificate is not visible anywhere in the Fastly user
+interface. It is managed entirely through Fastly support.
+
+Credentials for the Fastly Zendesk support site are in the [2nd line password store](https://github.com/alphagov/govuk-secrets/blob/master/pass/2ndline/fastly).
+
+## Renewing wildcard certificates
+
+Wilcard certificates for `*.publishing.service.gov.uk`, `*.staging.publishing.service.gov.uk`
+and `*.integration.publishing.service.gov.uk` are managed by AWS ACM.
+
+For AWS ACM to issue a certificate you must prove ownership of the domain using DNS.
```

Comment:
minor, missing comma: For AWS ACM to issue a certificate**,** you must prove ownership of the domain using DNS.